XCUITests/FxScreenGraph.swift

```diff
@@ -99,11 +100,18 @@ func createScreenGraph(_ app: XCUIApplication, url: String = "https://www.mozill
             }
 
             scene.tap(startBrowsingButton, to: NewTabScreen)
+            scene.tap(app.buttons["Sign in to Firefox"], to: FxSyncLogin)
```

Comment:
Ah I see, that makes totally sense. I have been looking at the code and looks like that button does not have acc.Id. Maybe I am wrong but have not seen it, in these logs I see the acc.Id for startBrowsingButton but not for Sing in: 

> Button 0x608000366fc0: traits: 8589934593, {{47.5, 462.0}, {280.0, 46.0}}, label: 'Sign in to Firefox'
> Button 0x608000367a40: traits: 8589934593, {{0.0, 611.0}, {375.0, 56.0}}, identifier: 'IntroViewController.startBrowsingButton', label: 'Start Browsing').

 I have tried to add this:

> signInButton.accessibilityIdentifier = "IntroViewController.signInButton"

Below this line https://github.com/mozilla-mobile/firefox-ios/blob/a15ada2e49fefbce30bd912fd8495606c1efb9f4/Client/Frontend/Intro/IntroViewController.swift#L165 in a similar way that acc.Id is defined for startBrowsingButton some [lines above](https://github.com/mozilla-mobile/firefox-ios/blob/041d04e2e7c7312256425687bccb4604c831c285/Client/Frontend/Intro/IntroViewController.swift#L100) 
The test passes that way, but not sure whether that is the solution, sorry for my ignorance on this, would need your input here @jhugman, thanks!